docs/src/management-commands.rst

```diff
@@ -101,6 +101,18 @@ Find version inconsistencies in the CMS::
 * ``MODEL``: The model to check (one of ``page``, ``event``, ``poi``)
 
 
+``hix_bulk``
+~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
``hix_bulk``
~~~~~~~~~~~~
```
Ah, sorry for not noticing this earlier.
Not sure why the doc tool isn't complaining... the underline should match the heading itself.